dev-support/atlas-docker/README.md

```diff
@@ -36,24 +36,31 @@ Docker files in this folder create docker images and run them to build Apache At
    ./download-archives.sh
    ~~~
 
-5.  Execute following commands to set environment variables to build Apache Atlas docker containers:
+5. Ensure that the `${HOME}/.m2` directory exists and has the correct ownership:
+   ~~~
+   mkdir -p ${HOME}/.m2
+   sudo chown -R $(whoami):$(whoami) ${HOME}/.m2
```

Comment:
`.m2/repository` directory can have large number of files and directories (> 50k files is not uncommon). Line # 42 can take a long time to complete.

Instead of inserting step #5 (and updating numbers of subsequent steps), I suggest to update 6.1 with inclusion of following line before starting build:

```
mkdir -p ${HOME}/.m2
```